seeds/dex/aggregator/dex_aggregator_seed.csv

```diff
@@ -97,13 +97,13 @@ ethereum,1inch,UNI v3,2022-11-12,0xb07dcf3cf82bb1dd1f9ab6da1562b1af01297e39c9a41
 polygon,bebop,2,2023-04-07,0x3107f005f3069e340d7316a5a6372cce5e0b2ba3c42eedc1f4e08e29903620c2,194,"0,0",0x7ceb23fd6bc0add59e62ac25578270cff1b9f619,0.16149840785059086,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,300
 arbitrum,bebop,2,2023-04-07,0xeec27114877fa983d5c8cbc9a5cb55dd966ad35e0d35ce30394ed8cd03faee12,12,"0,0",0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,152.13967,0x539bde0d7dbd336b79148aa742883198bbf60342,115
 ethereum,bebop,2,2023-04-07,0xd41613c9af3ef4adf84279f2b211a441aa3a37dd1c546d018ca6082911ccf743,24,"0,0",0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2,0.08539524191852751,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,170.855956
-ethereum,dodo,0,2022-10-30,0x24d3b64dca789018552b6f61eeed88632635dae60ec6762c437d61a57053eb90,285,,0xdac17f958d2ee523a2206206994597c13d831ec7,3271.009257,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,3271.850490
-bnb,dodo,0,2022-01-03,0x1d683cdcc7728b4f007c224aaa614e50b6a3ac42d37c7f2bd6ccc5e4f92355ab,178,,0x55d398326f99059ff775485246999027b3197955,10120.76926701322,0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d,10119.277349334472
-polygon,dodo,0,2021-06-25,0xbb50798e0ba6fe335e285f95345b7296a0adb09f24dd927ffd95b5c269d439cb,181,,0xc2132d05d31c914a87c6611c10748aeb04b58e8f,84.735894,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,84.721843
-arbitrum,dodo,0,2022-04-03,0xaaa1b052c5f1258e7625fbb384ee585b73cd636a514508e3254b8909d834e16d,20,,0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9,2252.765924,0xff970a61a04b1ca14834a43f5de4533ebddb5cc8,2253.326086
-optimism,dodo,0,2022-06-02,0x4326310124907a6c4c53fc8778d8bdcd30c3f06fa46edd72125cc4cfadcf9a14,5,,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,7.501047,0x7f5c764cbc14f9669b88837ca1490cca17c31607,7.49437
-base,dodo,0,2023-08-08,0xc19441e00c18a84963abbc09cabd84d49a6c64526eb67757592f8390d8c07229,66,,0xeb466342c4d449bc9f53a865d5cb90586f405215,1.877398,0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,0.001
-base,dodo,0,2023-10-07,0x6482fdaf8885d57d755adc907625529a22076bd370d8c0038ef7a1388b8559a9,33,,0xd9aaec86b65d86f6a7b5b1b0c42ffa531710b6ca,1.621019,0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee,0.001
+ethereum,DODO_X,0,2022-10-30,0x24d3b64dca789018552b6f61eeed88632635dae60ec6762c437d61a57053eb90,285,,0xdac17f958d2ee523a2206206994597c13d831ec7,3271.009257,0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48,3271.850490
```

Comment:
If you use `DODO X` in the model, then due to join condition, this would be `DODO X` not `DODO_X` as well?